ui/src/app/components/login-gov-ial2-notification.tsx

```diff
@@ -65,11 +67,17 @@ const styles = reactStyles({
 
 
 const LoginGovIAL2Notification = () => {
+  const [navigate, ] = useNavigation();
   return <FlexRow data-test-id='ial2-notification' style={styles.box}>
     <AlarmExclamation style={styles.icon}/>
     <div style={styles.text}>Please verify your identity by 10/06/2021.
     </div>
-    <Button type='primary' style={styles.button} onClick={() => location.replace('/?workbenchAccessTasks=true')}>
+    <Button
+        type='primary'
+        style={styles.button}
+        onClick={() => environment.enableDataAccessRequirements ?
+            navigate(['data-access-requirements']) :
+            location.replace('/?workbenchAccessTasks=true')}>
```

Comment:
It's because the homepage has 2 modes: registered and unregistered (w/ Dashboard).  This notification needs to appear on the homepage in the registered mode (we don't yet require RAS/IAL, so registered users typically don't have it).  `navigate()` does not work when it's the same page and we're only changing the queryParams.

Since DAR is its own page, we can exclude this notification from it, and therefore we always know it's navigating to a different page.